k2/csrc/rmepsilon_test.cc

```diff
@@ -0,0 +1,136 @@
+// k2/csrc/rmepsilon_test.cc
+
+// Copyright (c)  2020  Xiaomi Corporation (author: Haowen Qiu)
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/csrc/rmepsilon.h"
+
+#include <algorithm>
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "k2/csrc/fsa.h"
+#include "k2/csrc/fsa_equivalent.h"
+#include "k2/csrc/fsa_renderer.h"
+#include "k2/csrc/fsa_util.h"
+#include "k2/csrc/properties.h"
+
+namespace k2 {
+class RmEpsilonTest : public ::testing::Test {
+ protected:
+  RmEpsilonTest() {
+    std::vector<Arc> arcs = {
+        {0, 4, 1},        {0, 1, 1},        {1, 2, kEpsilon}, {1, 3, kEpsilon},
+        {1, 4, kEpsilon}, {2, 7, kEpsilon}, {3, 7, kEpsilon}, {4, 6, 1},
```

Comment:
FYI, I would have been fine with zeros here.  Either way.

Reply:
Agree with you  as it's just test code. Anyway, will not change back to zeros as either zeros or kEpsilon would be fine.